### DISCUSSION TITLE
Ansible "Failed to connect to the host via ssh: ssh

### DISCUSSION THREAD

**Discussion by vikram9491** [learndevopsonline/learndevopsonline#1564] [Q&A] [answered]
![Screenshot (81)](https://github.com/learndevopsonline/learndevopsonline/assets/67426874/cc8c6061-0245-4f9a-8d76-968618f1aad5)
hi @r-devops  getting this error even i have route53 records , saying service is unknown how to resolve it ...?
Frontend Ip : 54.224.103.58

**Comment by sreedharm07** [comment]
Hi
@vikram9491 error is because of dns records. 
i tried with ip its working
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/98fc1246-5c6c-45c1-b833-6e5e43782a15)

 and i tried catalogue and other components all records are not available .
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/8597b7ce-6dbb-4fd7-9092-db9d5fcee375)

**Reply by sreedharm07** [reply, marked as answer]
your dns was still not in not active in many locations. it seems you created hosted zone recently. it will take 24 hrs to active.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/381bc0b0-4156-4cb4-ba62-ffffdfb6d4db)

**Reply by vikram9491** [reply]
Hi @sreedharm07  still in some locations my domain name is not active , Previously i registered the same domain name in  another amazon account trial got expired for it payment not done , does this effect ..?

**Reply by sreedharm07** [reply]
@vikram9491 no it doesnt effect i think

**Reply by vikram9491** [reply]
@sreedharm07  should i deleted my hosted zone because still it is not active in many locations

vikramdevops.tech domain name checked with NS type record

**Reply by sreedharm07** [reply]
i think you created dns 3 days back but still not available in many locations, delete and create again bro
